migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,84 @@
+[id="migration-from-version-3-to-4-overview"]
+= Migration from version 3 to 4 overview
+
+{product-title} 4 clusters are different from {product-title} 3 clusters. {product-title} 4 clusters contain new technologies and functionality that result in a cluster that is self-managing, flexible, and automated To know more about migrating from {product-title} 3 to 4 see, xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[migrating from version 3 to 4].
+
+[discrete]
+[id="mtc-3-to-4-overview-differences-mtc"]
+=== Differences between {product-title} 3 and 4
+Differences between {product-title} 3 and 4: To successfully transition to {product-title} 4, review the following information:
+
+** Architecture
+** Installation and upgrade
+** Storage, network, logging, security, and monitoring considerations
+
+[discrete]
+[id="mtc-3-to-4-overview-planning-network-considerations-mtc"]
+=== Planning network considerations
+Planning network considerations: Before migrating from {product-title} 3 to 4 plan your network considerations, review the following information:
+
+** DNS considerations
+*** Isolating the DNS domain of the target cluster from the clients.
+*** Setting up the target cluster to accept the source DNS domain.
+
+You can migrate stateful application workloads from {product-title} 3 to 4 at the granularity of a namespace. To know more about MTC see the xref:../migration_toolkit_for_containers/about-mtc.adoc#about-mtc[Understanding MTC].
```

Comment:
These sentences and following note look a bit out of place. Where they are now is under the "networking considerations" section, but they look more generally applicable. Should they be moved somewhere else?

Also:
* To know more -> To learn more
* see the -> see